tests/unit/helpers/dates/companiDates.test.js

```diff
@@ -41,11 +50,62 @@ describe('CompaniDate', () => {
   });
 });
 
+describe('GETTER', () => {
+  describe('getUnits', () => {
+    const companiDate = CompaniDatesHelper.CompaniDate('2021-11-24T07:12:08.000Z');
+
+    it('should return units', () => {
+      const result = companiDate.getUnits(['day', 'second']);
+
+      expect(result).toEqual({ day: 24, second: 8 });
+    });
+
+    it('should return empty if unit is plural or invalid', () => {
+      const result = companiDate.getUnits(['days', 'second', 'mois']);
+
+      expect(result).toEqual({ second: 8 });
+    });
+  });
+});
+
 describe('DISPLAY', () => {
   describe('format', () => {
-    let _formatMiscToCompaniDate;
     const companiDate = CompaniDatesHelper.CompaniDate('2021-11-24T07:12:08.000Z');
```

Comment:
Qu'est-ce qui fait que tu definis cette constante en dehors du it ?

Reply:
une maladresse je pense. Je rectifie